Dockerfile

```diff
@@ -1,7 +1,20 @@
+FROM alpine:3.14 as build
+
+RUN apk --no-cache add \
+        curl \
+        bash \
+    && BABASHKA_VERSION=$(curl -sL https://raw.githubusercontent.com/borkdude/babashka/master/resources/BABASHKA_RELEASED_VERSION) \
+    && BABASHKA_ZIP=babashka-${BABASHKA_VERSION}-linux-static-amd64.zip \
```

Comment:
I'm not sure how this can work with the .zip extension. Shouldn't it be tar.gz and use tar to unpack it? All the latest releases have tar.gz, except for the windows release: https://github.com/babashka/babashka/releases

Reply:
Oh, I see, you are using the install script in the end, but I believe it would be better to do it this way: https://github.com/babashka/babashka#installer-script

```
$ curl -sLO https://raw.githubusercontent.com/babashka/babashka/master/install
$ chmod +x install
$ ./install
```